com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/UpdateServiceKeysStep.java

```diff
@@ -111,27 +115,29 @@ private boolean areServiceKeysEqual(CloudServiceKey key1, CloudServiceKey key2)
 
     private void deleteServiceKeys(CloudControllerClient client, List<CloudServiceKey> serviceKeys) {
         serviceKeys.stream()
```

Comment:
Creating a stream just to apply an operation to all elements in a collection is redundant.
Use a for each loop or Collection.forEach().

Reply:
fixed